src/routers/alpha-router/alpha-router.ts

```diff
@@ -1307,13 +1320,24 @@ export class AlphaRouter
           routingConfig,
           undefined,
           gasPriceWei
-        )
+        ).then((result) => {
+          metric.putMetric(
+            `SwapRouteFromCache_V2_GetRoutesAndQuotes_Load`,
+            Date.now() - beforeGetRoutesAndQuotes,
+            MetricLoggerUnit.Milliseconds
+          );
+
+          return result;
+        })
       );
     }
 
     if (mixedRoutes.length > 0) {
       const mixedRoutesFromCache: MixedRoute[] = mixedRoutes.map((cachedRoute) => cachedRoute.route as MixedRoute);
 
+      const beforeGetQuotes = Date.now();
+      metric.putMetric("SwapRouteFromCache_Mixed_GetQuotes_Request", 1, MetricLoggerUnit.Count);
```

Comment:
Aren't we missing this metric for V2? The request metric

Reply:
Good catch..